src/data-display/titles/page-title/story-helper.ts

```diff
@@ -205,4 +205,14 @@ export const getPageTitleArgTypes = (): ArgTypes => ({
             category: 'events',
         },
     },
+    clickTitle: {
+        name: 'clickTitle',
+        description: 'Emitted when title area is clicked',
+        table: {
+            type: {
+                summary: null,
+            },
+            category: 'events',
+        },
+    },
```

Comment:
suggestion: Our event name convention is kebab case :)
And please add 'event' postfix to event arg type's property name to prevent duplication of prop or slot arg type names! 😄

```suggestion
    clickTitleSlot: {
        name: 'click-title',
        description: 'Emitted when title area is clicked',
        table: {
            type: {
                summary: null,
            },
            category: 'events',
        },
    },
```

Reply:
Thanks for your suggestion! 
This feature would be develop through `template`, yuda suggested